content/reference/endpoints/bookings.md

```diff
@@ -33,20 +35,31 @@ source_id               | Integer | Write      | Source id related to the Bookin
+charge_damage_deposit_on_arrival  | Boolean    | Write | Defines if charge damage deposite will be charged on arrival.
 children                | Integer | Write      | Booking's number of children.
+comission               | [Decimal](/reference/enums#formats) | Write | Booking's comission fee.
 currency                | String  | Write      | Booking's currency code, list of codes described in [enums section](/reference/enums#currencies).
+damage_deposit          | [Decimal](/reference/enums#formats) | Write | Booking's damage deposit.
 discount                | String  | Write      | Booking's discount. (in percents e.g. 10%)
-downpayment             | [Decimal](/reference/enums#formats) | Write      | Booking's downpayment.
-final_price             | [Decimal](/reference/enums#formats) | Write      | Booking's final price. (after discount)
-initial_price           | [Decimal](/reference/enums#formats) | Write      | Booking's initial price.
+downpayment             | [Decimal](/reference/enums#formats) | Write | Booking's downpayment.
+expected_checkin_time   | String  | Write      | Expected guest's checkin time.
+expected_checkout_time  | String  | Write      | Expected guest's checkout time.
+final_payback_to_owner  | [Decimal](/reference/enums#formats) | Write | Final payback to rental owner.
+final_price             | [Decimal](/reference/enums#formats) | Write | Booking's final price. (after discount)
+initial_price           | [Decimal](/reference/enums#formats) | Write | Booking's initial price.
+locked                  | String  | Write      | Booking's locked status.
 notes                   | Text    | Write      | Booking's notes. (1000 characters max)
+reference               | String  | Read       | Booking's reference code.
+rental_payback_to_owner | [Decimal](/reference/enums#formats) | Write | Amount of money to pay back to owner (only rental, excluding fees).
 status                  | String  | Read       | Booking's status.
 unavailable             | Boolean | Write      | Booking's unavailable status, false by default. When set to true, marks the given period as unavailable. Commonly used for closed or maintenance periods.
 ------------------------|---------|------------|------------
 updated_at              | [Time](/reference/enums#formats) | Read       | Booking's update time.
 start_at                | [Time](/reference/enums#formats) | Read/Write | **Required**. Booking's start time.
 end_at                  | [Time](/reference/enums#formats) | Read/Write | **Required**. Booking's end time.
-tentative_expires_at    | [Time](/reference/enums#formats) | Write      | Booking's tentative expiry time, null by default. When set, it will mark the period as "on hold" until the given date. Once this date is passed, the period will automatically become available again.
+canceled_at             | [Time](/reference/enums#formats) | Write | Booking's cancel time.
```

Comment:
@Fredar  think the main idea behind this is that this scope will not return this parameter for all bookings, just for owned. If you look at the other params like children or adults, it's also only write. 
This is a bit messy, maybe we should somehow point out that for not-owned bookings it will display data as public scope, but for owned it will display same data as bookings_read/bookings_write?